node-red-contrib-ms-authentication/node-luis-config.html

```diff
@@ -32,6 +33,7 @@
                     $("#node-config-input-subKey").typedInput('show');
                     $("#node-config-input-verbose").typedInput('show');
                     $("#node-config-input-staging").typedInput('show');
+                    $("#node-config-input-spellCheckKey").typedInput('show');
```

Comment:
Actually, the spellcheck key can be configured only when using the "with keys" configuration mode. I think we should also allow it in the "with endpoint" configuration mode.